docs/hub/datasets-duckdb.md

```diff
@@ -0,0 +1,21 @@
+# DuckDB
+
+[DuckDB](https://github.com/duckdb/duckdb) is an in-process SQL OLAP database management system.
+Since it supports [fsspec](https://filesystem-spec.readthedocs.io) to read and write remote data, you can use the Hugging Face paths (`hf://`) to read and write data on the Hub:
+
+First login using 
+
+```
+huggingface-cli login
+```
+
+And then you can use Hugging Face paths in DuckDB:
+
+```python
+>>> from huggingface_hub import HfFileSystem
+>>> import duckdb
+
+>>> fs = HfFileSystem()
+>>> duckdb.register_filesystem(fs)
+>>> df = duckdb.query(f"SELECT * FROM 'hf://datasets/username/my_dataset/data.parquet' LIMIT 10").df()
+```
```

Comment:
this is only to read. Add a command to (create and) write?

Reply:
it doesn't seem to work

```
In [5]: duckdb.sql("COPY data TO 'hf://datasets/lhoestq/tmp-duckdb-export/data.parquet' (FORMAT PARQUET);")
Exception ignored in: <function AbstractBufferedFile.__del__ at 0x10548b1f0>
Traceback (most recent call last):
  File "/Users/quentinlhoest/.pyenv/versions/3.9.15/envs/hf-datasets/lib/python3.9/site-packages/fsspec/spec.py", line 1966, in __del__
    self.close()
  File "/Users/quentinlhoest/.pyenv/versions/3.9.15/envs/hf-datasets/lib/python3.9/site-packages/fsspec/spec.py", line 1944, in close
    self.flush(force=True)
  File "/Users/quentinlhoest/.pyenv/versions/3.9.15/envs/hf-datasets/lib/python3.9/site-packages/fsspec/spec.py", line 1815, in flush
    if self._upload_chunk(final=force) is not False:
  File "/Users/quentinlhoest/.pyenv/versions/3.9.15/envs/hf-datasets/lib/python3.9/site-packages/huggingface_hub/hf_file_system.py", line 425, in _upload_chunk
    path_in_repo=self.resolved_path.path_in_repo,
AttributeError: 'HfFileSystemFile' object has no attribute 'resolved_path'
```

cc @mariosasko @Wauplin

Reply:
Actually I had to create the repository and it worked. Opened https://github.com/huggingface/huggingface_hub/issues/1800 to improve the error message

Reply:
added a code snippet to create a repo and write using DuckDB